docs/source/logging/logging.md

```diff
@@ -1,54 +1,121 @@
+# Default logging configuration
+Kedro's [default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/default_logging.yml) defines a handler called `rich` that uses the Rich logging handler to format messages. We also use the Rich traceback handler to render exceptions.
 
-# Default framework-side logging configuration
+By default, Python only shows logging messages at level `WARNING` and above. Kedro's logging configuration specifies that `INFO` level messages from Kedro should also be emitted. This makes it easier to track the progress of your pipeline when you perform a kedro run.
```

Comment:
I would make this sentence the first one in the file, and then follow it with "Kedro's [default logging configuration](...)..."